jsons/Eras.json

```diff
@@ -0,0 +1,212 @@
+[
+    {
+        "name": "Arrival era",
+        "researchAgreementCost": 350,
+        "startingSettlerCount": 3,
+        "startingWorkerCount": 2,
+        "startingMilitaryUnitCount": 5,
+        "startingMilitaryUnit": "Rifleman",
+        "startingGold": 200,
+        "startingCulture": 500,
+        "settlerPopulation": 3,
+/*        "settlerBuildings": ["Monument","Granary","Lighthouse","Market","Workshop","Amphitheater","Barracks","Library","Colosseum"],
```

Comment:
We might want to keep these buildings, as they will still work as long as they have entries in Buildings.json. Otherwise new cities might be pretty weak. If we add new buildings, then disregard.